### PR TITLE
require nvfuser version 0.2.8

### PR DESCRIPTION
**Pull request by jjsjann123**
What's in this PR:
* requiring nvfuser version 0.2.8 for inplace patch;
* removing code for old nvfuser API change.

### CODE REVIEW

**Comment by jjsjann123** [comment]
marking this as draft to ensure that we have proper nvfuser packages published and available in CI.

**Comment by t-vi** [comment]
Marking as ready as the conditions seem to apply.

**Review by t-vi — APPROVED** [review]
Thank you @jjsjann123

File: thunder/executors/nvfuserex.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def nvfuser_version() -> LooseVersion | None:
 
 
 def required_nvfuser_version() -> LooseVersion:
```
Comment (IvanYashchuk) [inline thunder/executors/nvfuserex.py:32]: If nvFuser is installed but the version is not new enough should Thunder issue a warning about this?

Comment (t-vi) [reply thunder/executors/nvfuserex.py:32]: I think an error would be also good.

**Pull request merged**